saltyrtc/server/events.py

```diff
@@ -0,0 +1,31 @@
+import collections
+from enum import (
```

Comment:
Ah. Alright, I'll change that. I won't do it for `typing` values though, as in that case the signatures would get really long.